assignments/arrays.js

```diff
@@ -63,17 +63,21 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 1999, Jeep, Wrangler`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length -1];
+console.log(lastCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = []
+for (let i =0; i < inventory.length; i++){
+    let model = inventory[i].car_model
+    {model.sort}
```

Comment:
`sort` is a function.  So we want to call it with `array.sort()`. We also want to call it on the array of car models, with `carModels.sort()`. We also want to do it outside of the for loop once we are done adding all of the models instead of after we add each one. Both would work, but it is inefficient to do it for every iteration of the for loop.